morango/models.py

```diff
@@ -1,75 +1,108 @@
+import json
+
 from django.db import models
 
-from .utils.uuids import UUIDModelMixin
-from .settings import INDEX_FIELDS
+from .utils.uuids import UUIDModelMixin, UUIDField
 
 
 ###################################################################################################
-# METADATA: Additional data to keep track of the state of the system (sessions, counters, etc)
+# APP MODELS: Abstract models from which app models should inherit in order to make them syncable
 ###################################################################################################
+class SyncableModelQuerySet(models.query.QuerySet):
 
+    def update(self, **kwargs):
+        kwargs.update({'_dirty_bit': True})
+        super(SyncableModelQuerySet, self).update(**kwargs)
+    update.queryset_only = True  # Unsure whether django will not place this on manager class by default
```

Comment:
Agreed. I can't remember why I did it this way but yeah I think it should also be on the manager.